pkg/apis/pipeline/v1beta1/results.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I should've made this clearer in the task breakdown summary - we are introducing these changes in v1alpha first. Then we gradually migrate that to v1beta. I updated this requirement in https://github.com/tektoncd/pipeline/issues/4723#issue-1186799973

Reply:
Correct me if I'm wrong. I thought we have discussed this in chat with @jerop  as well. v1alpha has different fields than v1beta1, they are quite different and I think the TEP is based on apis in v1beta1. 

in alpha1:
https://github.com/tektoncd/pipeline/blob/7e5e4c0f7b50c40192d46153580bfb1f38891a1b/pkg/apis/pipeline/v1alpha1/task_types.go#L116-L133

in beta1:
https://github.com/tektoncd/pipeline/blob/7e5e4c0f7b50c40192d46153580bfb1f38891a1b/pkg/apis/pipeline/v1beta1/task_types.go#L130-L137

Reply:
Ah you are right about that we are adding the changes in apiversion v1beta1 and keep it as alpha features. Resolving this comment.